doc/api/buffer.md

```diff
@@ -273,7 +272,7 @@ function:
 
 * [`Buffer.from(array)`]
 * [`Buffer.from(buffer)`]
-* [`Buffer.from(arrayBuffer[, byteOffset [, length]])`][`Buffer.from(arrayBuffer)`]
+* [`Buffer.from(arrayBuffer[, byteOffset[, length]])`][`Buffer.from(arrayBuf)`]
```

Comment:
Same as the above case: Probably don't want to make this change within this PR.

Reply:
ditto.